app/containers/App/reducer.js

```diff
@@ -24,8 +26,13 @@ const initialState = fromJS({
 
 const appReducer = (state = initialState, action) => {
   switch (action.type) {
+    case LOAD_TOKEN:
+      const lsToken = ls.get('token');
```

Comment:
You shouldn't read the token from local storage here, in the reducer. The reducers should be pure functions (google it :) ). You read the local storage in the action function (you can use thunk middleware) and then dispatch another action that gets processed here, in the reducer, in order to set the token in the state.